Tests/SentryTests/Integrations/Performance/UIViewController/SentryTimeToDisplayTrackerTest.swift

```diff
@@ -0,0 +1,218 @@
+import Foundation
+import Sentry
+import SentryTestUtils
+import XCTest
+
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+
+class SentryTimeToDisplayTrackerTest: XCTestCase {
+
+    private class Fixture {
+        let dateProvider: TestCurrentDateProvider = TestCurrentDateProvider()
+        var tracer: SentryTracer {  SentryTracer(transactionContext: TransactionContext(operation: "Test Operation"), hub: nil) }
+
+        var displayLinkWrapper = TestDisplayLinkWrapper()
+
+        func getSut(for controller: UIViewController, waitForFullDisplay: Bool) -> SentryTimeToDisplayTracker {
+            let framesTracker = SentryFramesTracker(displayLinkWrapper: displayLinkWrapper)
+            framesTracker.start()
+            return SentryTimeToDisplayTracker(for: controller, frameTracker: framesTracker, waitForFullDisplay: waitForFullDisplay)
+        }
+    }
+
+    private let fixture = Fixture()
+
+    override func setUp() {
+        super.setUp()
+        CurrentDate.setCurrentDateProvider(fixture.dateProvider)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        clearTestState()
+    }
+
+    func testreportInitialDisplay_notWaitingFullDisplay() throws {
```

Comment:
`l`: Please also update the other test cases.
```suggestion
    func testReportInitialDisplay_notWaitingFullDisplay() throws {
```